frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -95,8 +76,31 @@ common_info.report_bug = {
         else
             msg = common_msg
         end
-        UIManager:show(InfoMessage:new{
+        UIManager:show(ConfirmBox:new{
             text = msg,
+            no_cancel_button = true,
+            ok_text = _("Toggle verbose logging"),
```

Comment:
Well, I won't see that ConfirmBox often, and so won't most people. So, is does not need to be pretty nor ugly - but it would be nice if it still has a `Cancel` button like most/all others, just so people not familiar with dissmis by tap outside don't get there wondering how to get out, and taping the only thing they think they can tap: Toggle verbose logging :)

ConfirmBox can have 2 wide buttons (in case you removed Cancel because it was taking room needed by the wide "Toggle verbose logging":
https://github.com/koreader/koreader/blob/e0ed04a1a147162c0c03e5864df2e62e7a5b9ba2/frontend/ui/widget/confirmbox.lua#L52-L53

Reply:
Is question icon needed? (no questions in the text)
Maybe, ButtonDialog? Without changes to the ConfirmBox widget.

Reply:
The cancel button for dismissal was included in my suggestion, but the fact that ConfirmBox comes with an icon is something I wasn't thinking of. :-)

Btw, I'm not sure how much sense a checkmark makes there in the menu?

Reply:
> ConfirmBox can have 2 wide buttons (in case you removed Cancel because it was taking room needed by the wide "Toggle verbose logging":

Oh, I looked at that feature *before* coming up with removing the Cancel, and then forget about it ;p. Two rows of wide buttons, with cancel at the bottom would probably be okay, yeah.

> Is question icon needed?

That's a fair point. I honestly don't even notice the icons anymore in those popups ;p.

> Maybe, ButtonDialog? 

I thought about that, but thought I'd have to reinvent the wheel a bit too much for that one.

> Btw, I'm not sure how much sense a checkmark makes there in the menu?

The idea is to have *something* to show that verbose debug logging is enabled at a glance.
I'm not super happy about it, but so far I find that it works better compared to the only other solution I could think of, which was a [`text_func`](https://github.com/koreader/koreader/pull/11366#discussion_r1454340103).

Reply:
> > Maybe, ButtonDialog?
> 
> I thought about that, but it looked like I'd have to reinvent the wheel a bit too much for that one.

Specifically, that I'd have to stick some kind of text widget to it to actually display the most important part of the original and main intent of it ;).